code/style.css

```diff
@@ -1,4 +1,310 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/*MOBILE (max-width=667px) */
+
+body {
+  min-width: 500px;
+  background: white;
+  font-family: helvetica;
+  margin: 0;
 }
+
+/*Header*/
+header {
+  width: 100%;
+  background: rgba(247, 222, 0, 0.959);
+  height: 150px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-left: 20px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: white;
+  border-radius: 5px;
+}
+
+h1 {
+  font-size: 50px;
+  color: white;
+  margin: 30px;
+}
+
+.logo {
+  display: flex;
+  width: 130px;
+  justify-content: center;
+  margin: 20px;
+}
+
+nav {
+  display: none;
+  margin: 0px 40px;
+}
+
+nav a {
+  display: none;
+  color: white;
+  text-decoration: none;
+  margin-right: 25px;
+  font-size: 20px;
+}
+
+/*Big News*/
+.container {
+  background: rgb(212, 83, 93);
+  padding: 40px 0px;
+  width: auto;
+  border-radius: 5px;
+  margin: 30px;
+  min-width: 400px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.container img {
+  border-radius: 5px;
+  width: 400px;
+}
+
+.container div {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 20px;
+  color: white;
+}
+
+h2 {
+  font-size: 25px;
+}
+
+h3 a {
+  font-size: 20px;
+  text-decoration: none;
+  color: white;
+}
+
+/*Other News*/
+.other-news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.container-two {
+  position: relative;
+  border-radius: 5px;
+  box-shadow: 5px 5px 0 0 rgba(207, 73, 84, 0.705);
+  margin: 30px 100px;
+  max-width: 400px;
+}
+
+.container-two img {
+  display: flex;
+  border-radius: 5px;
+}
+
+.text-block {
+  position: absolute;
+  bottom: 10px;
+  background-color: rgba(212, 83, 94, 0.692);
+  font-size: 25px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+/*Footer*/
+footer {
+  background: rgb(212, 83, 93);
+  width: 100%;
+  height: 150px;
+}
+
+h4 {
+  font-size: 20px;
+  color: white;
+  display: flex;
+  justify-content: center;
+  padding: 40px;
+}
+
+
+/*** MEDIA QUERIES ***/
+
+/*** TABLET ***/
+@media (min-width: 668px) and (max-width: 1023px) {
+
+  body {
+    min-width: 720px;
+  }
+
+  /*Header*/
+  header {
+    width: auto;
+    background: rgba(247, 222, 0, 0.959);
+    height: 150px;
+    display: flex;
```

Comment:
Since your first CSS code (everything written above line 151) doesn't have a media query, you can think of that as the default styling. That will apply all the time, unless you change it. Therefore you don't have to specify the same properties again, e.g. `display: flex;` is already set to the `header`, so you only have to include it in the media queries if you want to change the value of it.